config/config.go

```diff
@@ -147,12 +147,16 @@ func (c Configuration) New(
 		jaeger.TracerOptions.ZipkinSharedRPCSpan(opts.zipkinSharedRPCSpan),
 	}
 
+	for _, tag := range opts.tags {
+		tracerOptions = append(tracerOptions, jaeger.TracerOptions.Tag(tag.Key, tag.Value))
+	}
```

Comment:
Yeah, did the exact same thing, but I guess this crept on while resolving some conflicts. Will fix it.